test/user/user_test.dart

```diff
@@ -76,10 +76,10 @@ Future main() async {
     expect(subreddits.length, equals(1));
     final subreddit = subreddits[0];
     expect(subreddit.displayName, equals('drawapitesting'));
-    expect(await subreddit.isContributor, isTrue);
-    expect(await subreddit.isBanned, isFalse);
-    expect(await subreddit.title, equals('DRAW API Testing'));
-    expect(await subreddit.property('public_description'),
+    expect(subreddit.isContributor, isTrue);
+    expect(subreddit.isBanned, isFalse);
+    expect(subreddit.title, equals('DRAW API Testing'));
+    expect(subreddit.data['public_description'],
```

Comment:
Does the use of `contains` here follow the specification or are we being lazy here?